app.js

```diff
@@ -35,7 +35,7 @@ var mybundle = gpClient.bundle(bundleName);
 // Example: /
 app.get('/', function (req, res) {
   // For the root page, load the list of supported langs 
-  mybundle.getInfo({}, function (err, projInfo) {
+  mybundle.getInfo(function (err, projInfo) {
```

Comment:
in v1.3.0, where  the `{}` options are ignored or optional, they may be omitted.